### PR TITLE
Add Hover Behavior

### PR DESCRIPTION
**Pull request by DireCorgi**
### Changes
- Add hover for buttons
- Add hover for logo
- Add hover for cocktail cards

### Trello
- https://trello.com/c/sRNt2zsY/1-style-hovers

### CODE REVIEW

File: src/assets/styles/_variables.scss [hunk 1]
```diff
@@ -16,14 +16,23 @@ $napkin-path:   '/images/imgborderbg-black.png';
...
+$light-navy: lighten($navy, 10%);
 $amethyst: #330928;   // vodka
+$light-amethyst: lighten($amethyst, 10%);
 $russet: #6f3d08;     // whiskey
+$light-russet: lighten($russet, 10%);
 $marigold: #a79112;   // (rum)
-$black: #000;         // other
+$light-marigold: lighten($marigold, 10%);
+$black: #111;         // other
+$light-black: lighten($black, 10%);
```
Comment (diiiefiend) [inline src/assets/styles/_variables.scss:34]: please make 10% a variable so it's easy to change

File: src/assets/styles/components/cocktail-box.scss [hunk 2]
```diff
@@ -6,9 +6,15 @@
   min-height: 200px;
 
   h3 {
+    color: $black;
```
Comment (diiiefiend) [inline src/assets/styles/components/cocktail-box.scss:9]: is this still explicitly needed after having `color` defined on the body?

File: src/components/CocktailBox.vue [hunk 3]
```diff
@@ -1,10 +1,17 @@
 <template>
-  <div class="cocktail-box">
+  <div
+    class="cocktail-box"
+    @mouseover="hovered = true"
+    @mouseleave="hovered = false"
+  >
     <router-link to="/cocktail">
-      <h3>
+      <h3 :class="{ hovered: hovered }">
```
Comment (diiiefiend) [inline src/components/CocktailBox.vue:8]: does Vue support object shorthand here? so `:class="{ hovered }"`?

File: src/assets/styles/components/cocktail-box.scss (hunk 2 above)
Comment (DireCorgi) [reply src/assets/styles/components/cocktail-box.scss:9]: Yes, colors are getting inherited from a

File: src/components/CocktailBox.vue (hunk 3 above)
Comment (DireCorgi) [reply src/components/CocktailBox.vue:8]: yes

File: src/assets/styles/_variables.scss (hunk 1 above)
Comment (DireCorgi) [reply src/assets/styles/_variables.scss:34]: okay

File: src/assets/styles/components/cocktail-box.scss (hunk 2 above)
Comment (diiiefiend) [reply src/assets/styles/components/cocktail-box.scss:9]: i will look into that

**Review by diiiefiend — APPROVED** [review]
LGTM, thanks for the work! 🛸

**Pull request merged**